docs/content/reference/credentials_collectors/mimikatz.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Mimikatz"
+date: 2023-09-13T16:51:44+05:30
+tags: ["credentials collector", "mimikatz", "windows"]
+weight: 2
+---
+
+## Description
+
+The Mimikatz Credentials Collector uses [mimikatz](https://github.com/gentilkiwi/mimikatz)
```

Comment:
Don't we use "pypykatz", though? Does pypykatz actually use mimikatz under the hood, or does it reimplement the functionality natively in Python?

Reply:
Good catch, it's pure python